libs/angular/src/lib/dropdown/dropdown.component.ts

```diff
@@ -125,6 +135,14 @@ export class NggDropdownComponent
 
   writeValue(value: any): void {
     this.value = value
+    if (
+      value === null &&
+      this.dropdown?.isTouched &&
+      this.control?.control?.untouched
+    ) {
+      this.setHandler()
```

Comment:
Do we need to create and set the handler again or can we update the current one? I'm out on a limb here :)

Reply:
i was thinking about that also, if wouldnt be any side effects to change only `handler.dropdown.isTouched` property  on `formControl.reset()`

Any thoughts from others?

Reply:
Yes that sounds like a good idea, it would not have any side effects - but make sure to change the condition on line 160 in green\libs\extract\src\lib\dropdown\dropdown.ts to only trigger when the new value is true.

Also don't modify the isTocuhed property directly, create a new reducer in extracts and make it callable. Let me know if you need any clarification or help :)

Reply:
thanks, updated PR with suggestions, i hope didnt miss anything now.